salt/version.py

```diff
@@ -96,107 +96,107 @@ class SaltStackVersion(object):
         'Nitrogen'      : (MAX_SIZE - 102, 0),
         'Oxygen'        : (MAX_SIZE - 101, 0),
         'Fluorine'      : (MAX_SIZE - 100, 0),
+        'Neon'          : (MAX_SIZE - 99, 0),
```

Comment:
Yeah, we do need the `Neon` line uncommented for this change for the `warn_until`. But git is being silly by changing _all_ of those lines...